test/chi/components/avatar.pug

```diff
@@ -1,8 +1,8 @@
 ---
 title: Avatar
 ---
-- var types = ['img', 'icon', 'icon-web-font', 'letter', 'double-letter'];
-- var colors = ["default", "grey", "red", "pink", "purple", "indigo", "blue", "light-blue", "teal", "mint", "green", "yellow", "orange", "light", "secondary", "primary", "inverse"];
```

Comment:
@sabah1910 my comment about removing light and inverse versions just made reference in avatar group example as we currently don't support this use case. But it is supported in the other avatar examples without text beside it. Please add light and inverse in the var colors again but remove it from the example in avatar groups

Reply:
@jllr I fixed this. However, for the secondary avatar group circle, I updated it so that Label secondary words are in the same line and this made all the circles uniform size. But when I minimize the screen the circle becomes oval for both secondary and light-blue. Please advise.